src/Components/IngredientDetails/IngredientDetails.css

```diff
@@ -1,81 +1,188 @@
-.details-wrapper {
-  /* border: 2px solid purple; */
-  height: 85vh;
-  display: flex;
-  align-items: center;  
-  justify-content: center; 
-  font-family: 'HyliaSerifBeta-Regular';
-  color: white;
-}
+@media screen and (min-width: 950px) {
 
-.details-container {
-  background-color: rgb(0, 0, 0, .7);
-  height: 51vh;
-  width: 55%;
-  border-radius: 2%;
-  padding-bottom: .2em;
-  box-shadow: 0px 0.2em 0.5em #536f50;
-  /* box-shadow: 0px 0.2em 0.5em #7EF0F4; */
+  .details-wrapper {
+    /* border: 2px solid purple; */
```

Comment:
I'm sure this'll be addressed in a refactor branch, but there's a few commented out properties in most of these files whenever that branch happens!